src/pybind/mgr/tox.ini

```diff
@@ -70,6 +70,39 @@ setenv =
 passenv =
     MYPYPATH
 basepython = python3
+mypy_args =
```

Comment:
@adk3798 hi Adam, i'd suggest revert this change.

Reply:
yeah, I noticed this when opening backports for your patches (https://github.com/ceph/ceph/pull/49361) and at least  kept this part out of the backports. Reversion PR is now up https://github.com/ceph/ceph/pull/49375. My mistake.